translation-client/src/main/java/eu/europeana/api/translation/client/utils/TranslationClientUtils.java

```diff
@@ -46,6 +50,44 @@ public static Function<UriBuilder, URI> buildUrl(String path) {
         };
     }
 
+    /**
+     * Creates LangDetectRequest from LanguageDetectionObj
+     * @param languageDetectionObjs
+     * @return
+     */
+    public static LangDetectRequest createLangDetectRequest(List<LanguageDetectionObj> languageDetectionObjs) {
+        LangDetectRequest langDetectRequest = new LangDetectRequest();
+        // hint is optional
+        String hint = languageDetectionObjs.get(0).getHint();
+        if (StringUtils.isNotEmpty(hint)) {
+            langDetectRequest.setLang(hint);
+        }
+        List<String> text = new ArrayList<>(languageDetectionObjs.size());
+        for (LanguageDetectionObj object : languageDetectionObjs) {
+            text.add(object.getText());
+        }
+        langDetectRequest.setText(text);
+        return langDetectRequest;
+    }
+
+    /**
+     * Creates TranslationRequest from TranslationObj
+     * @param translationStrings
+     * @return
+     */
+    public static TranslationRequest createTranslationRequest(List<TranslationObj> translationStrings) {
+        TranslationRequest translationRequest = new TranslationRequest();
```

Comment:
this looks more like a factory method which can be implemented directly in the class that fires the request. There is no reason to have this as a static method